docs/version0.12.1.md

```diff
@@ -0,0 +1,37 @@
+<!--
+* Copyright (c) 2017, 2019 IBM Corp. and others
```

Comment:
Start copyright should be 2019.

Reply:
We view the doc as a collection, therefore the copyright is the same across all files regardless of when they were created.